src/fn_call.rs

```diff
@@ -341,13 +491,27 @@ pub trait EvalContextExt<'a, 'mir, 'tcx: 'a + 'mir>: crate::MiriEvalContextExt<'
                 //     data_ptr: *mut usize,
                 //     vtable_ptr: *mut usize,
                 // ) -> u32
-                // We abort on panic, so not much is going on here, but we still have to call the closure.
                 let f = this.read_scalar(args[0])?.to_ptr()?;
                 let data = this.read_scalar(args[1])?.not_undef()?;
+                let data_ptr = this.deref_operand(args[2])?;
+                let vtable_ptr = this.deref_operand(args[3])?;
                 let f_instance = this.memory().get_fn(f)?;
                 this.write_null(dest)?;
                 trace!("__rust_maybe_catch_panic: {:?}", f_instance);
 
+                // In unwind mode, we tag this frame with some extra data.
+                // This lets '__rust_start_panic' know that it should jump back
+                // to this frame is a panic occurs.
```

Comment:
That is implemented in `pop_stack`, right? If yes please say so.

Reply:
```suggestion
            // to this frame if a panic occurs.
```

Reply:
Yes, that's correct.

Reply:
Okay. Please add a comment in the code.